jsontokens/key_loading.py

```diff
@@ -44,11 +42,9 @@ def __init__(self):
 def load_signing_key(signing_key, crypto_backend=default_backend()):
```

Comment:
That's fine; it's just that `Multibackend` is not a module we can import any longer.